src/prefect/testing/standard_test_suites/blocks.py

```diff
@@ -28,3 +29,15 @@ def test_all_fields_have_a_description(self, block: Type[Block]):
             assert field.field_info.description.endswith(
                 "."
             ), f"{name} description on {block.__name__} does not end with a period"
+
+    def test_has_a_valid_code_example(self, block: Type[Block]):
+        code_example = block.get_code_example()
+        assert code_example is not None, f"{block.__name__} is missing a code example"
+        import_pattern = rf"from .* import {block.__name__}"
+        assert (
+            re.search(import_pattern, code_example) is not None
+        ), f"The code example for {block.__name__} is missing an import statement"
+        block_load_pattern = rf'.* = {block.__name__}\.load\("BLOCK_NAME"\)'
+        assert re.search(
+            block_load_pattern, code_example
+        ), f"The code example for {block.__name__} is missing a .load statement"
```

Comment:
Strange thought, but I was wondering if we need tests for testing these standard test suites?

Reply:
I don't think so. They're tests and we're running them against blocks in the code base, so we can't treat them like any other test.